android/KMAPro/kMAPro/src/main/res/values/strings.xml

```diff
@@ -13,14 +13,14 @@
   <string name="action_settings" comment="Menu action to open Settings menu">Settings</string>
   <string name="action_install_updates" comment="Menu notification that keyboard or dictionary updates available">Install Updates</string>
 
-  <string name="title_version">Version: %1$s</string>
+  <string name="title_version" comment="Title of Keyman for Android version">Version: %1$s</string>
 
   <!-- &#8230; is the ellipsis character "..." -->
-  <string name="textview_hint">Start typing here&#8230;</string>
+  <string name="textview_hint" comment="Prompt to start typing">Start typing here&#8230;</string>
 
   <!-- Splash screen (version/copyright info currently disabled -->
```

Comment:
Note that this will show as part of context for "splash_blank" but not "splash_copyright". Same for all XML comments. It's a bit annoying but suggest we either have XML comments for each entry with more details (like what I did for windows/src/desktop/kmshell/xml/strings.xml) or eliminate them and use whitespace to separate groups.

Reply:
I'll use the XML comments to provide additional context for each string